options/locale/locale_si-LK.ini

```diff
@@ -2117,7 +2117,7 @@ dashboard.task.finished=කාර්යය:%[1]s[2]s විසින් ආර
 dashboard.task.unknown=නොදන්නා කාර්යය:%[1]s
 dashboard.cron.started=ආරම්භ Con:%[1]s
 dashboard.cron.process=ක්රෝන්:%[1]s
-dashboard.cron.cancelled=Con: %s අවලංගු:%[3]ගේ
```

Comment:
@wxiaoguang is'nt that missing a type (s)

Reply:
Yes, this key in en-US has changed, so it triggers re-translation, the `dashboard.cron.cancelled` won't exist in main branch IMO